tests/integration_tests.rs

```diff
@@ -235,6 +236,10 @@ async fn test_wasi_provider() -> Result<(), Box<dyn std::error::Error>> {
                         {
                             "mountPath": "/bar",
                             "name": "configmap-test"
+                        },
+                        {
+                            "mountPath": "/baz",
```

Comment:
Right now this is just checking that we can open the dir and give it to the running wasmtime module. I opened #205  to capture the need for making a separate module